app/models/user.rb

```diff
@@ -30,9 +31,25 @@ class User < ApplicationRecord
             format: { with: URI::MailTo::EMAIL_REGEXP, message: 'must be an email address' }
 
   validates :password,
-            length: { minimum: 8 }
+            length: { minimum: 8, allow_nil: true }
+
+  validates :settings,
```

Comment:
:monkas: can't at least validate that it's an option?

Reply:
Looks like even a raw string in the body like `"foo"` will get parsed into `{"_json"=>"foo"}` so don't think there's much point.